src/components/details/Details.js

```diff
@@ -0,0 +1,144 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useParams } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faArrowLeft, faCity, faGear, faMicrophone,
+} from '@fortawesome/free-solid-svg-icons';
+import Moment from 'moment';
+import { fetchCountry } from '../../redux/countries/countries';
+import formatNumber from '../utils/formatNumber';
+import image from '../../assets/virus.svg';
+import './Details.css';
+
+const Details = () => {
+  const { name } = useParams();
+  const dispatch = useDispatch();
+  const { country, loading } = useSelector((state) => ({
+    loading: state.loadingBar.default,
+    country: state.countries.selected,
+  }));
+
+  useEffect(() => {
+    dispatch(fetchCountry(name));
+    // fetchCountry(name);
+  }, []);
+
+  if (loading || !country) {
+    return null;
+  }
+
+  const { All } = country;
+  const list = Object.entries(country).slice(1);
+
+  if (Object.keys(country).length > 1) {
+    return (
+      <section>
+        <header className="App-header">
+          <Link to="/">
+            <FontAwesomeIcon icon={faArrowLeft} />
+          </Link>
+          <h5 className="App-header-title">view Detailed</h5>
+          <FontAwesomeIcon icon={faMicrophone} />
+          <div className="pl-5">
+            <FontAwesomeIcon icon={faGear} />
+          </div>
+        </header>
+        <div className="Details-banner">
+          <div className="Details-banner-left">
+            <img src={image} alt="map view" className="App-map" />
+          </div>
+          <div className="Details-banner-right">
+            <h1 className="App-title">{All.country}</h1>
```

Comment:
* Your details page doesn't work as expected i get a blank page whenever i click on a particular country and errors in the console too. Kindly fix this 🙏 . Below is a screenshot of the said error ⬇️ 
![image](https://user-images.githubusercontent.com/52497246/181135452-f872ff00-0243-40dc-9c25-e32d895a190e.png)

Reply:
### @Davidosky007 

From my side everything is working correctly on local. as you can see in the screenshot.

So I don't understand the issue

![Screenshot (150)](https://user-images.githubusercontent.com/40032163/181190228-080b6092-18de-41bd-ac69-0eb244a21d60.png)
![Screenshot (149)](https://user-images.githubusercontent.com/40032163/181190262-b2d27328-d781-4908-bf2b-41c59189e321.png)